### ISSUE TITLE
ModuleNotFoundError: No module named 'paralleltask'

### ISSUE THREAD

**Issue by WengangXbio** [Nextomics/NextDenovo#211]
**Question or Expected behavior**
I got an error when submit nextdenovo as a job, say"ModuleNotFoundError: No module named 'paralleltask'"

**Operating system**
LSB Version:
Distributor ID: CentOS
Description:    CentOS Linux release 7.9.2009 (Core)
Release:        7.9.2009
Codename:       Core

**GCC**
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/libexec/gcc/x86_64-redhat-linux/4.8.5/lto-wrapper
Target: x86_64-redhat-linux
Configured with: ../configure --prefix=/usr --mandir=/usr/share/man --infodir=/usr/share/info --with-bugurl=http://bugzilla.redhat.com/bugzilla --enable-bootstrap --enable-shared --enable-threads=posix --enable-checking=release --with-system-zlib --enable-__cxa_atexit --disable-libunwind-exceptions --enable-gnu-unique-object --enable-linker-build-id --with-linker-hash-style=gnu --enable-languages=c,c++,objc,obj-c++,java,fortran,ada,go,lto --enable-plugin --enable-initfini-array --disable-libgcj --with-isl=/builddir/build/BUILD/gcc-4.8.5-20150702/obj-x86_64-redhat-linux/isl-install --with-cloog=/builddir/build/BUILD/gcc-4.8.5-20150702/obj-x86_64-redhat-linux/cloog-install --enable-gnu-indirect-function --with-tune=generic --with-arch_32=x86-64 --build=x86_64-redhat-linux
Thread model: posix
gcc version 4.8.5 20150623 (Red Hat 4.8.5-44) (GCC) 

**Python**
Python 3.12.3

**NextDenovo**
nextDenovo 2.5.2

**Additional context** (Optional)
I can run example case successfully, while came across this problem when submit as a job.
I check my python packages, there is paralleltask as follow
(nextdenovo) [yzwl_zhangwg@mgt19 nextdenovo]$ pip list 
Package      Version
------------ -------
Paralleltask 0.1.1
pip          24.0
psutil       5.9.8
setuptools   70.0.0
wheel        0.43.0

would be appreciate if you can help me on this, many thanks!

**Comment by moold** [comment]
This is usually caused by the compute nodes and the master node using different Python programs. 
Try: 
```
`which python` nextDenovo run.cfg
```

**Comment by WengangXbio** [comment]
Hi developer, 
Thank you for helping me on this, now the tool can move forward. But I came across another error reported as [" returned non-zero exit status 134, error info"].  

![image](https://github.com/Nextomics/NextDenovo/assets/85557809/04e563f0-62ee-4367-9ae2-9f5266fb912e)

 I saw there are 65 jobs in total  for raw align step, 59 jobs failed, and the program try to rerun the other failed for three times. I guess the failure is due to the full of RAM and thread, right?  
![image](https://github.com/Nextomics/NextDenovo/assets/85557809/dfcf4dc2-c18e-4752-85ed-c3da2a44c9f9)

The program is still running, I monitoring its CUP and RAM, CPU is 100% in use and RAM is 80% , I assume most of jobs will still fail after three times attempts, right?

my code is pasted as below, which parameters I can optimize to reduce CPU usage for passing this step? parallel_jobs  or minimap2's "-t", I am confused with this two parameters. THank you very much! Looking forward to get assembly from this tool!
```
job_type = local
job_prefix = nextDenovo
task = all # 'all', 'correct', 'assemble'
rewrite = yes # yes/no
deltmp = yes
rerun = 3
parallel_jobs = 12
input_type = raw
read_type = ont
input_fofn = ./input.fofn
workdir = ./01_rundir

[correct_option]
read_cutoff = 3k
genome_size = 3100000000
pa_correction = 10
sort_options = -m 300g -t 12 -k 27
minimap2_options_raw =  -t 12 -x ava-ont
correction_options = -p 15

[assemble_option]
minimap2_options_cns =  -t 12
nextgraph_options = -a 1
```
